pal/src/misc/bstr.cpp

```diff
@@ -130,16 +130,20 @@ STDAPI_(BSTR) SysAllocStringLen(const OLECHAR *psz, UINT len)
  *  return value = void
  *
  ***********************************************************************/
-STDAPI_(void) SysFreeString(const OLECHAR* psz)
+STDAPI_(void) SysFreeString(BSTR bstr)
 {
-    if (psz != NULL)
+    if (bstr != NULL)
     {
-        HeapFree(GetProcessHeap(), 0, (LPVOID) psz);
+        bstr = (BSTR) ((char*) bstr - sizeof(DWORD));
+#if defined(_WIN64)
+        bstr = (BSTR) ((char*) bstr - sizeof(DWORD));
```

Comment:
> update: so maybe we should also remove 99 to 106 too ?

I thought about the same, but didn't make the change since we are not using this on _WIN64 anyway. Left it there in case one day we found we need it for Linux x64.